webapp/blueprint.py

```diff
@@ -70,18 +70,18 @@ def config():
             "config.js.jinja",
             apiAddress=JIMM_WSS_URL,
             baseUrl="/",
-            bundleServiceURL="https://" + JAAS_API_BASE + "/bundleservice",
-            charmstoreURL="https://" + JAAS_API_BASE + "/charmstore",
+            bundleServiceURL=JAAS_API_BASE + "/bundleservice",
```

Comment:
It might be nice to append a protocol to `JAAS_API_BASE` in this file if it doesn't include one in .env. But this could be a followup.